Content.Shared/Slippery/SlipperySystem.cs

```diff
@@ -18,6 +21,8 @@ public sealed class SlipperySystem : EntitySystem
 {
     [Dependency] private readonly ISharedAdminLogManager _adminLogger = default!;
     [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly INetManager _netManager = default!;
+    [Dependency] private readonly SharedPopupSystem _popup = default!;
```

Comment:
if you remove the pop up you don't need this?

Reply:
I have removed it